### ISSUE TITLE
Tests don't work with pytest >= 4

### ISSUE THREAD

**Issue by mnhauke** [easysnmp/easysnmp#121]
Support for calling fixtures directly was removed in pytest 4.0

python2
```
============================= test session starts ==============================
platform linux2 -- Python 2.7.17, pytest-4.6.9, py-1.8.1, pluggy-0.13.1
rootdir: /home/abuild/rpmbuild/BUILD/easysnmp-0.2.5, inifile: setup.cfg
plugins: sugar-0.9.2, flake8-1.0.4, cov-2.8.1
collected 50 items / 2 errors / 48 selected

==================================== ERRORS ====================================
_____________________ ERROR collecting tests/test_easy.py ______________________
Fixture "sess_v1_args" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.
____________________ ERROR collecting tests/test_session.py ____________________
Fixture "sess_v1" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.

```

python3
```
============================= test session starts ==============================
platform linux -- Python 3.8.2, pytest-4.6.9, py-1.8.1, pluggy-0.13.1
rootdir: /home/abuild/rpmbuild/BUILD/easysnmp-0.2.5, inifile: setup.cfg
plugins: cov-2.8.1, flake8-1.0.4, sugar-0.9.2
collected 42 items / 2 errors / 40 selected

==================================== ERRORS ====================================
_____________________ ERROR collecting tests/test_easy.py ______________________
Fixture "sess_v1_args" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.
____________________ ERROR collecting tests/test_session.py ____________________
Fixture "sess_v1" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.
```

**Comment by kamakazikamikaze** [comment]
Will fix with Github Actions integration. Thanks for the report.

**Comment by kamakazikamikaze** [comment]
Tests now function properly in `actions-test` branch. Will merge to `master` once CI/CD is fully functional.

I'll keep the issue open until a new package release is uploaded.

**Comment by kamakazikamikaze** [comment]
Fixed with #136
